controller/templates/statefulset.yaml

```diff
@@ -177,4 +179,26 @@ spec:
             limits:
               cpu: 100m
               memory: 32Mi
+        {% else %}
+        - name: passthrough-proxy
+          image: traefik:v2.5
+          args:
+           - --entryPoints.web.address=:4180
+           - --providers.file.directory=/traefik
+           - --log.level=DEBUG
```

Comment:
I think this should not be the default here as it makes for heavy logs and increases the risk of accidentally logging some credentials.

Reply:
```suggestion
```